src/client/components/SignupComponent/index.scss

```diff
@@ -0,0 +1,75 @@
+$bluecolor: #4A90E2;
+$greencolor: #36A420;
+$redcolor: #be4b49;
+
+html, body {
```

Comment:
this things are already set in the src/styles.scss dont need to define them again